src/components/storage/Common.jsx

```diff
@@ -128,3 +139,17 @@ export const useMountPointConstraints = () => {
 
     return mountPointConstraints;
 };
+
+export const getNewPartitioning = async ({ currentPartitioning, method = "AUTOMATIC" }) => {
+    await setInitializeLabelsEnabled({ enabled: true });
```

Comment:
I am wondering if this should be true only for AUTOMATIC.
In TUI we have this rule: https://github.com/rhinstaller/anaconda/blob/a73fe25bfb521fbf59f24b991a83cd5e9c2cb9d4/pyanaconda/ui/tui/spokes/storage.py#L521
(but in TUI the alternative partitioning is only MANUAL).
In GUI seemingly not but I am not sure:  https://github.com/rhinstaller/anaconda/blob/a73fe25bfb521fbf59f24b991a83cd5e9c2cb9d4/pyanaconda/ui/gui/spokes/storage.py#L226
(maybe it is harmless for other cases)

In the end the method feels more like getNewAutomaticPartitioning at this point, but perhaps it can be used for other types of partitioning in the future.

Anyway, I wouldn't block the PR on clearing up this, I'd need to figure out myself how this works exactly for various use cases.